### PR TITLE
Ac 901 jammy images

### PR DESCRIPTION
**Pull request by ru-fu**
- Update references to bionic and focal images with references to jammy images
- Update the requirements to support jammy for the full deployment (not the appliance)

### CODE REVIEW

**Review by morphis — APPROVED** [review]
LGTM, thanks!

File: requirements.md [hunk 1]
```diff
@@ -70,11 +48,46 @@ For external access to the Anbox Cloud Appliance, a couple of network ports must
...
+<a name="ac-ubuntu-version"></a>
+### Ubuntu version
+
+Anbox Cloud supports the following Ubuntu versions:
+
+* 18.04 (bionic) - see [Linux kernel](#linux-kernel) below for limitations
+* 20.04 (focal)
+* 22.04 (jammy)
+
+For new deployments, Ubuntu 22.04 (jammy) is preferred.
```
Comment (adglkh) [inline requirements.md:72]: For now, our CI only covers the following Ubuntu distros for the spread tests against deployment-scripts
- bionic
- focal

I don't think we run spread tests on the jammy for the deployment-scripts yet, @morphis Could you confirm this?

**Comment by ru-fu** [comment]
Split this into two PRs now:
- This one to update image references to Jammy.
- https://github.com/anbox-cloud/discourse-docs/pull/137 to support/recommend Jammy for installations.

**Comment by adglkh** [comment]
LGTM

**Pull request merged**